doc/02-usage.md

```diff
@@ -3,7 +3,7 @@
 After [installing](./00-installation.md) `otk` you'll probably want to start using it. If you've followed the [installation instructions](./00-installation.md) for a source checkout you'll have an examples directory available. You can compile one of the examples like so:
 
 ```
-€ otk compile examples/fedora/minimal-40-x86_64.yaml
+€ otk compile example/fedora/fedora-40-x86_64-minimal-raw.yaml
```

Comment:
It's _probably_ better at this point to have this point at the centos example. They're much more complete.

However I might also spend some time to get Fedora Minimal working correctly so let's keep this for now :)

Reply:
It at least compiles, you just can't run it through osbuild. Which is when I discovered it isn't really like the others :)